helpers/helpers.go

```diff
@@ -93,3 +93,21 @@ func PersistExistingParams(values []string, key, ignoreValue string, q url.Value
 func ToBoolPtr(val bool) *bool {
 	return &val
 }
+
+// GetOfficialStatisticsLogo returns the official statistics logo based on the enableOfficialStatisticsLogo and language
+func GetOfficialStatisticsLogo(enableOfficialStatisticsLogo bool, language string, useSvg bool) string {
```

Comment:
Go saves you the trouble of writing the same type each time if they are the same
i.e. 
```suggestion
func GetOfficialStatisticsLogo(enableOfficialStatisticsLogo, useSvg bool, language string) string {
```